models/comment_model.go

```diff
@@ -1,10 +1,23 @@
 package models
 
-import "google.golang.org/genproto/googleapis/type/date"
+import (
+	"time"
+
+	"cloud.google.com/go/firestore"
+)
 
 type Comment struct {
-	Comment_id int       `json:"comment_id"`
-	Content    string    `json:"content"`
-	Like       string    `json:"like,omitempty"`
-	Date       date.Date `json:"date"`
+	Content string                   `json:"content"`
+	Like    []*firestore.DocumentRef `json:"like,omitempty"`
+	Date    time.Time                `json:"date,omitempty"`
+}
+
+type CommentResponse struct {
+	UserId       string    `json:"userId"`
+	PostId       string    `json:"postId"`
+	Comment_id   string    `json:"comment_id"`
+	Content      string    `json:"Content"`
+	CountComment int       `json:"countComment"`
+	Likec        string    `json:"like,omitempty"`
+	Date         time.Time `json:"date,omitempty"`
```

Comment:
1. (line 18): don't want field **Comment_id** 
2. (line 20 ): don't want field **CountComment** 
3. (line 21): you spell Like wrong, pls edit and renaming to `CountLike`

Reply:
จากข้อ 2 ให้มองว่า 1 model comment = 1 comment 
พอเป็น 1 comment -> มันจะไม่สามารถ return countComment ได้ 

ในทางกลับกัน มองแบบ post model = 1 post 
ถ้าเป็น model post มันจะ return เป็น countComment ได้